cloudslang-compiler/src/test/resources/flow_with_external_steps_and_errors.sl

```diff
@@ -0,0 +1,58 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
+#   All rights reserved. This program and the accompanying materials
+#   are made available under the terms of the Apache License v2.0 which accompany this distribution.
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+namespace: user.ops
+
+imports:
+  ops: user.ops
+
+flow:
+  name: flow_with_external_steps_and_errors
+  workflow:
+    - DoNothing:
+        do_external:
```

Comment:
this is an erronous case, right? or are we supporting such syntax? do we have a test for it?